src/Symfony/Component/Profiler/ProfileData/MemoryData.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+
+namespace Symfony\Component\Profiler\ProfileData;
+
+
+class MemoryData implements ProfileDataInterface
+{
+
+    protected $memory;
```

Comment:
please use the private visibility everywhere. We always make things private by default as it makes the maintenance much easier (everything protected involves some BC concern as it becomes an extension point)